src/main/java/com/amazon/inspector/jenkins/amazoninspectorbuildstep/AmazonInspectorBuilder.java

```diff
@@ -230,9 +235,12 @@ public void perform(Run<?, ?> build, FilePath workspace, EnvVars env, Launcher l
 
             if (componentName.endsWith(".tar")) {
                 sanitizedImageId = sanitizeFilePath("file://" + componentName);
```

Comment:
It may be worth renaming `sanitizedImageId` for clarity.
Consider `sanitizedArtifactName` perhaps?

Reply:
Good idea.